### ISSUE TITLE
Select All based actions

### ISSUE THREAD

**Issue by ellisonbg** [jupyterlab/jupyterlab#3564]
Right now if we a user wants to apply an action to all cells, we have separate "All" versions of commands, along with associated menu items, commands, etc. I am wondering if it makes sense to instead have a single "Select All" command that the user can use to apply actions/commands to all cells. This would simplify the UI dramatically and provide 1 way of doing things. Not sure this makes sense, just thinking out loud.

@ian-r-rose @jasongrout let's talk to the designers about this.

**Comment by ian-r-rose** [comment]
We do currently have a `Select All Cells` command in the edit menu. At some point (though I don't recall exactly when), we discussed this in our meetings with the Bloomberg UX team. That is, we could cut down on many of the `... All` commands if they are paired with selecting all. Bonnie and Paddy were, IIRC, not too wild about the idea, since many common actions would go from one user action to two.

I think it's fair to say that at least some of the `... All` commands are common enough that we should keep them (`Run All Cells`, `Clear All Cells`, etc). Others, such as expanded and collapsing related commands, may be more marginal.

**Comment by ellisonbg** [comment]
@ian-r-rose thank for the background!

I was thinking more about this while running errands. In visualization, there is an emphasize on "grammar", which means giving users smaller bits of functionality that have rules about how they can be composed. I think think this is a great example of that. One effect of embracing Select All is that it completely solves the keyboard shortcut questions for the "All" actions.

Marking as "Needs Discussion"

**Comment by ian-r-rose** [comment]
Yes, my instinct would be towards smaller, more composable actions as well. In the short term, I can't think of any reason not to have `Accel A` map to `Select All` when in command mode.

It's worth noting that the further we go (especially after we start recommending JupyterLab to end users), the more difficult it will be to take away menu items.

**Comment by jasongrout** [comment]
Ping on this beta 2 issue. It sounds like we've had discussion now. Any clear resolution in people's minds?
